slices.go

```diff
@@ -150,3 +157,20 @@ func SliceCopyFn[T any](s []T,
 
 	return result
 }
+
+// SliceRandom returns a random element from a slice
+func SliceRandom[T any](a []T) T {
+	var result T
+
+	switch len(a) {
+	case 0:
+		return result
+	case 1:
+		result = a[0]
+	default:
+		rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
shouldn't we _assume_ someone already initialised `rand.Reader` before calling `SliceRandom()`?

Reply:
we do on `acmefy/pkg/ca` ....

Reply:
We could be calling this function a lot, and initialising a new source every this time sounds... excessive

Reply:
that might be true. Let me think about it.